### ISSUE TITLE
Provide quick possibility to add the following episodes of a series to the same hour in the next days

### ISSUE THREAD

**Issue by scr0llbaer9000** [TVTower/TVTower#1185]
There has been a request in the forum to fill the next days in the program planner with the episodes of a series with ~one click.

https://www.gamezworld.de/phpforum/viewtopic.php?pid=90577#p90577

Of course, problems arise if something is already scheduled at that spot in one of the next days. Should the existing programme be "overwritten", or should the series just make a pause for that day and continue next day (as it often is in real life when there are special events, like live sports etc). Implementing a dialog to ask the player what they want in that case could be a bit cumbersome.

**Comment by nittka** [comment]
You can already navigate to the next day with "page down", so placing episodes at the same time every day is quite easy (without moving the mouse) while retaining complete control of what is happening.

The most trivial implementation would be "Alt Click places episodes of a series in the same time slot for the following days"
* Version a: stop if placing the eposiode would remove an existing programme (relevant in particular for 2-block episodes or a series where not every episode has the same length)
* Version b: place the episode in any case; existing programmes are removed

I think only the second variant makes sense. Once the episodes are placed, you cannot remove all of them easily (e.g. if you started with the wrong episode or want to broadcast a different series instead).

**Comment by nittka** [comment]
Another option would be only moving to the next day on pressing Alt. This would leave control for the user and placing a series over several days would still be much easier.

**Comment by nittka** [comment]
"Alt" is already used for showing trailer sheet (not really needed in my opinion) as well as fixing the audience by target group tool tip (could probably be guarded by checking if the mouse is in the interface area).

Even disabling those two, dropping a programme is not possible while Alt is down. With Shift+Ctrl it works, but the visual feedback is not really nice. The next day is opened but you do not see that the placement for the previous day worked...

**Comment by nittka** [comment]
prototype branch is pushed...

**Comment by nittka** [comment]
I have played around a bit more: as page up/down may not be available, I checked whether +/- could be used for navigation. However, Blitzmax does not provide a key code for the regular plus (only for the number key pad which will usually not by available conveniently for notebooks).

My suggestion is to allow arrow key up and down for changing by exactly one day. In combination with shift, this will allow an easier placement of episodes over several days.

Apart from the implementation effort, at leaset the following arguments speak agains auto-placement
* It is not guaranteed that everybody in all circumstances wants to place episodes on consecutive days in the same time slot. (sport programmes, broadcast only Monday-Friday; different weekend programme; show two episodes per day)
* Handling existing placements (remove/stop) makes things complicated (user interaction, undo functionality).
* Control loss (check that you really want to place the programme - topicality not high enough)

A simple way to change days without moving the mouse is enough to improve usability while allowing full flexibility for placing the programmes.
